nixos/modules/services/misc/sourcehut/service.nix

```
@@ -108,7 +108,7 @@ let
       #SocketBindDeny = "any";
       SystemCallFilter = [
         "@system-service"
-        "~@aio" "~@keyring" "~@memlock" "~@privileged" "~@resources" "~@timer"
+        "~@aio" "~@keyring" "~@memlock" "~@privileged" "~@timer"
```

I believe this is a shorter way to say the same thing.

```suggestion
        "~@aio @keyring @memlock @privileged @timer"
```

---

This can be done in a separate PR. I'm not willing to introduce _even more_ invasive changes here at this point, just to be sure.